input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -321,28 +271,21 @@ For instance, suppose we want to create a table of all patient and contact addre
 }
 ```
 
-The above example uses `forEach` to select parts of the resource for each select to be unioned, but other use cases may get the needed columns directly
-in the select. See the [PatientAndContactAdddressUnion example](Binary-PatientAndContactAddressUnion.html) for a complete version of the above.
+The above example uses `forEach` to select different data elements from the resources to be unioned. For other use cases, it is possible to define the columns directly
+in the `select`. See the [PatientAndContactAdddressUnion example](Binary-PatientAndContactAddressUnion.html) for a complete version of the above.
```

Comment:
Suggest "one or more constants" rather than "one or more of constants".

Reply:
Good catch. Fixed!